src/cljs/netrunner/stats.cljs

```diff
@@ -7,12 +7,16 @@
             [netrunner.deckbuilder :refer [process-decks num->percent]]
             [netrunner.auth :refer [authenticated] :as auth]
             [netrunner.ajax :refer [POST GET]]
+            [netrunner.ws :as ws]
             [goog.string :as gstring]
             [goog.string.format]))
 
-(def stats-channel (chan))
-;(def stats-socket (.connect js/io (str js/iourl "/stats")))
-;(.on stats-socket "netrunner" #(put! stats-channel (js->clj % :keywordize-keys true)))
+(ws/register-ws-handler!
+  :stats/update
+  #(go (let [result (-> (<! (GET "/user")) :json first :stats)
```

Comment:
The websocket can send the new stats back with the `:stats/update` message, instead of the client needing to do two more requests for the data. When sending `:stats/update` in the server, the parameter that follows is the payload message. Don't send the string `"update"`, instead send a vector or a map with the two things that you need client side.